docs/development.md

```diff
@@ -30,15 +30,16 @@
 ### Base requirements
 
 1. Install [go][go]
-   - Get the latest patch version for go v1.11.
+   - Get the latest patch version for go v1.12.5.
```

Comment:
Tiny not here: specify `v1.12` instead so that the docs survive all versions of go 1.12